docs/synapse/userguides/storm_ref_lift.rstorm

```diff
@@ -322,450 +405,821 @@ Storm supports a set of extended comparison operators (comparators) for speciali
+.. storm-pre:: inet:ipv4:asn *in= ( 9009, 20473, 44477 )
+::
+  
+  inet:ipv4:asn *in= ( 9009, 20473, 44477 )
 
+*Lift all tags (syn:tag nodes) whose final tag element matches any of the specified string values:*
 
-.. storm-pre:: [file:bytes=sha256:00ecd10902d3a3c52035dfa0da027d4942494c75f59b6d6d6670564d85376c94 :size=2000]
-.. storm-opts:: {"show": ["err", "fini", "init", "warn"]}
-.. storm-cli:: file:bytes:size*range=(1000,100000)
+.. storm-pre:: [ syn:tag=rep.talos.plugx syn:tag=rep.eset.korplug syn:tag=rep.mandiant.sogu syn:tag=rep.alienvault.kaba ]
+.. storm-pre:: syn:tag:base *in= ( plugx, korplug, sogu, kaba )
+::
+  
+  syn:tag:base *in= ( plugx, korplug, sogu, kaba )
 
 
-- Lift WHOIS records that were captured between November 29, 2013 and June 14, 2016:
+.. _lift-proximity:
 
+Lift by Proximity (\*near=)
++++++++++++++++++++++++++++
 
-.. storm-pre:: [inet:whois:rec=(marsbrother.com, 2013/12/02 00:00:00.000)]
-.. storm-opts:: {"show": ["err", "fini", "init", "warn"]}
-.. storm-cli:: inet:whois:rec:asof*range=(2013/11/29, 2016/06/14)
+The proximity extended comparator (``*near=``) supports lifting nodes by "nearness" to another node. Currently, ``*near=``
+supports proximity based on geospatial location (i.e., nodes within a given radius of a specified latitude / longitude).
 
+**Syntax:**
 
-- Lift DNS requests made within one day of 12/01/2018:
+*<form>* **:** | **.** | **:_** *<prop>* ***near = ((** *<lat>* **,** *<long>* **),** *<radius>* **)**
 
+**Examples:**
 
-.. storm-pre:: [inet:dns:request="*" :time="2012/03/03 21:09:04.000"]
-.. storm-opts:: {"show": ["err", "fini", "init", "warn"]}
-.. storm-cli:: inet:dns:request:time*range=(2018/12/01, "+-1 day")
+*Lift all locations (geo:place nodes) within 500 meters of the Russian Cryptographic Museum (where the coordinates 55.83069,37.59781 represent the Museum's location):*
 
+.. storm-pre:: [ ( geo:place="*" :latlong="55.83088, 37.59962" :name="concern avtomatika" ) ]
+.. storm-pre:: geo:place:latlong *near= ( (55.83069, 37.59781), 500m )
+::
+  
+  geo:place:latlong *near= ( (55.83069, 37.59781), 500m )
 
 **Usage Notes:**
 
-- When specifying a range, both the minimum and maximum values are included in the range (the equivalent of "greater than or equal to *<min>* and less than or equal to *<max>*").
-- When specifying a range of time values, Synapse allows the use of both lower resolution values (e.g., ``YYYY/MM/DD``) and wildcard values (e.g., ``YYYY/MM*``) for the minimum and/or maximum range values. In addition, plain wildcard time syntax may provide a simpler and more intuitive means to specify some time ranges. For example ``inet:whois:rec:asof=2018*`` is equivalent to ``inet:whois:rec:asof*range=('2018/01/01', '2018/12/31 23:59:59.999')``.  See the type-specific documentation for :ref:`type-time` types for a detailed discussion of these behaviors.
+- Radius can be specified in the following metric units:
```

Comment:
We do support feet/yards/miles now as well (full list at https://github.com/vertexproject/synapse/blob/master/synapse/models/geospace.py#L10-L36)